File: es_06.cpp
// 6. Inserisci 10 numeri,
// calcolare il modulo (matematico) di ogni numero,
// fare la media dei numeri inseriti.

#include <iostream>
using namespace std;

int main() {
    int n, i, somma = 0;
    float media;

    for (i = 1; i <= 10; i++) {
        cout << "Inserisci un numero: ";
        cin >> n;

        if (n < 0) {
            n *= -1;
        }

        somma += n;
    }

    media = float(somma) / i;   // media = static_cast<float>(somma) / i;


    cout << "La somma è: " << somma << endl
         << "La media è: " << media << endl;

    return 0;
}

File: Python/es_06.py
from itertools import count

# 6. Inserisci 10 numeri,
# calcolare il modulo (matematico) di ogni numero,
# fare la media dei numeri inseriti.

somma = 0

for i in count (start=1, step=1):
    n = int(
        input(
            'Inserisci un numero: '
        )
    )

    print(
        f'Il modulo è {abs(n)}', end='\n\n'
    )

    somma += n

    if i == 10: break

print(
    f'La media è: {somma / i}'
)